BUILD.md

```diff
@@ -119,7 +121,7 @@ make docs
 
 To view the results, open `build/docs/html/index.html` in a web browser.
 
-### Python Lib
+### BitBox02 Python Library
```

Comment:
uuh didn't the other commit just remove title case? :D

Reply:
Yes, but only with the last commit, this one was earlier… 😇

Reply:
Indeed!